risc0/zkvm/sdk/rust/methods/test-resources/simple-c-pkg/depend/foo_rel.h

```diff
@@ -0,0 +1,6 @@
+#ifndef RISC0_FOO_REL_H
```

Comment:
This file looks identical to the dev version. Are we trying to test that debug vs release builds work? Wouldn't we want to ensure this feature works on the same code?